controllers/storagecluster/reconcile.go

```diff
@@ -172,6 +172,13 @@ func (r *StorageClusterReconciler) Reconcile(ctx context.Context, request reconc
 		return reconcile.Result{}, err
 	}
 
+	// Ensure that verbose logging or debugging is enabled when RBD mirroring is enabled
+	// This is a temporary arrangement, this is to be removed when RDR goes to GA
+	err = r.ensureRbdMirrorDebug(sc)
+	if err != nil && !errors.IsNotFound(err) {
```

Comment:
Just curious, why is this error ignored here?  What resource is expected to be not found?

Reply:
Thanks for noticing, This is a mistake left from me. I meant to ignore the is already exists error, But seems like I was confused. Will correct this.

Reply:
You should be ignoring the AlreadyExist error, not the notFound here right?

Reply:
Yes, I will set it to ignore the already exists error instead of this not found error